.github/workflows/push-release.yml

```diff
@@ -74,14 +78,5 @@ jobs:
         env:
           REPOSITORY: ghcr.io/${{ steps.repo_name.outputs.repository }}
         run: |
-          # Get artifacts from previous steps
-          docker pull ${{ steps.docker_meta.outputs.tags }}-amd64
-          docker pull ${{ env.REPOSITORY }}:stable-amd64
-          # Create and update manifests
-          docker manifest create ${{ steps.docker_meta.outputs.tags }} ${{ steps.docker_meta.outputs.tags }}-amd64
-          docker manifest annotate ${{ steps.docker_meta.outputs.tags }} ${{ steps.docker_meta.outputs.tags }}-amd64 --arch amd64
-          docker manifest create ${{ env.REPOSITORY }}:stable ${{ env.REPOSITORY }}:stable-amd64
-          docker manifest annotate ${{ env.REPOSITORY }}:stable ${{ env.REPOSITORY }}:stable-amd64 --arch amd64
-          # push manifests
-          docker manifest push ${{ steps.docker_meta.outputs.tags }}
-          docker manifest push ${{ env.REPOSITORY }}:stable
+          docker buildx imagetools create -t ${{ env.REPOSITORY }}:stable ${{ env.REPOSITORY }}:stable-amd64
```

Comment:
it is possible to specify multiple target tags by providing multiple -t arguments. 
I think it is possible to push required tags with single docker buildx call
```
docker buildx imagetools create -t ${{ env.REPOSITORY }}:stable \
-t ${{ steps.docker_meta.outputs.tags }} \
${{ env.REPOSITORY }}:stable-amd64
```